docs/csharp/misc/cs0119.md

```diff
@@ -19,18 +19,20 @@ ms.assetid: 048924f1-378f-4021-bd20-299d3218f810
 - A method identifier is used as if it were a struct or class  
   
 ## Example  
- The following sample generates CS0119.  
+ The following sample generates CS0119: 'C.B()' is a method, which is not valid in the given context 
```

Comment:
I think this needs more context/explanation rather than giving just the code.

@BillWagner Is this case stated somewhere in the language spec. That would be a good reference here.

Reply:
This error is related to the rules on binding, and overload resolution. Those rules get complicated very quickly, so I wouldn't put a reference here. See me other review comment.